packages/flutter/lib/analysis_options_user.yaml

```diff
@@ -62,6 +62,8 @@ linter:
     - package_api_docs
     - package_names
     - package_prefixed_library_names
+    - prefer_const_constructors
```

Comment:
This will cause lots of new warnings for existing users when they upgrade their Flutter version.

Reply:
That is a fair point. I was leaning toward more stringent linting but you're right -- we might alarm people unnecessarily.

Reply:
Agreed. Removed.